src/@types/types.d.ts

```diff
@@ -190,6 +190,79 @@ export interface GroupItem {
 
 export type FlagTagsFilter = Record<string, Record<string, boolean | GroupItem>>;
 
+export type GlobalFilterWorkloads = {
```

Comment:
Put these variables from the store in types, let me know with a better place. :sweat_smile: